gulpfile.js

```diff
@@ -0,0 +1,18 @@
+var gulp = require('gulp');
```

Comment:
uhm...Is gulp and browsersync really necessary?(both tools are nice, but I believe it's overkill for a simple library)

So, why don't use webpack over npm scripts https://github.com/zenorocha/clipboard.js/blob/master/package.json#L43-L49 instead?